### PR TITLE
[PUBDEV-7106] Report full flatfile in h2odriver

### PR DESCRIPTION
**Pull request by jakubhava**
Just proposal

### CODE REVIEW

**Comment by michalkurka** [comment]
lgtm - I would just fix the handling of the file resource but apart of that - I don't see an issue

**Comment by jakubhava** [comment]
Perfect, let me have a look on this! Thank you!

File: h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java [hunk 1]
```diff
@@ -586,6 +599,7 @@ public void registerNode (String ip, int port, int attempt, Socket s) {
           flatfile += val;
           flatfile += "\n";
         }
+        createFlatFile(flatfile);
```
Comment (michalkurka) [inline h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java]: I would probably move this to somewhere higher level after the clouding is successfully completed.

If everything succeeds `flatfileContent` is available in a static field

Comment (jakubhava) [reply h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java]: Good idea, done

**Comment by jakubhava** [comment]
@michalkurka please have a look. I have moved the creation & ensured we handle the IO in the same way as the cluster notification file

**Comment by jakubhava** [comment]
Thanks for the changes @michalkurka, are we good to go?

**Comment by michalkurka** [comment]
@jakubhava almost - I will need to run Hadoop smoke tests on this branch

I will merge when they pass

**Comment by jakubhava** [comment]
Thank you!

**Pull request merged**